include/Core/UI/Setting/EmailLogSetting.php

```diff
@@ -0,0 +1,113 @@
+<?php
+namespace EmailLog\Core\UI\Setting;
+use EmailLog\Core\UI\Setting\SettingField;
+
+class EmailLogSetting extends Setting {
+
+	/**
+	 * Implement `initialize()` method.
+	 */
+	protected function initialize() {
+		$this->section->id          = 'email-log';
+		$this->section->title       = __( 'Plugin Settings', 'email-log' );
+		$this->section->option_name = 'el_email_log';
+
+		$this->load();
+	}
+
+	/**
```

Comment:
The phpdoc for all the functions in this class needs to be changed.

If we are not adding any new information in the derived class's phpdoc block then we don't have to add a new phpdoc block

Reply:
Removed the doc block.